src/honeybot/plugins/downloaded/fact/main.py

```diff
@@ -39,8 +39,7 @@ def run(self, incoming, methods, info, bot_info):
     + "Instead, they played piano music."
     "Amelia Earhart and Eleanor Roosevelt once sneaked out of a White"
     + "House event, commandeered an airplane, and went on a joyride to Baltimore."
-    "The 100 folds in a chef's toque are said to represent 100 ways "
```

Comment:
There's no need to use `+` if it is a single string

Reply:
> There's no need to use `+` if it is a single string

These "+" was to obey the 100 char limit. I guess that new lint pass modified the original flake8 style